gddo-server/template.go

```diff
@@ -121,7 +122,13 @@ func (pdoc *tdoc) addExamples(obj interface{}, export, method string, examples [
 		id += "-" + method
 	}
 	for _, e := range examples {
-		te := &texample{Label: label, ID: id, Example: e, obj: obj}
+		te := &texample{
+			Label:        label,
+			ID:           id,
+			Example:      e,
+			ShowPlayLink: e.Play != "" && gosrc.IsGoRepoPath(pdoc.ImportPath),
```

Comment:
You're doing more complex logic here that is more disconnected (i.e., located further away) from the context where it's actually used.

I suggest adding a new field `Standard`, to mimic [`Standard` in `cmd/go`](https://github.com/golang/go/blob/28ce6f3600fd87a1ae39c492ee56307f0be3c32f/src/cmd/go/pkg.go#L40) (or maybe `Goroot` instead) here, and set its value independently.